src/task_queue/events_nudge/cadmin_events_nudge.py

```diff
@@ -205,7 +205,8 @@ def send_events_nudge():
 
 def send_events_report(name, email, event_list):
     try:
-        change_preference_link = ADMIN_URL_ROOT+"/admin/profile/preferences"
+        cred = encode_data_for_URL({"email": email})
```

Comment:
should this also include the login_method (or could that login_method always be added if an email is specified)?

Reply:
the login_method can be included.

Reply:
If you have the utility routine add that may simplify the usage